src/components/Testimonial/index.js

```diff
@@ -0,0 +1,26 @@
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class Testimonial extends Component {
+  static propTypes = {
+    image: PropTypes.string,
+    body: PropTypes.string,
+    quotee: PropTypes.string,
+  };
+
+  render() {
+    const { image, body, quotee, classes } = this.props;
```

Comment:
Is there a reason this is quotee and not quote lol

Reply:
I believe it's the person who was being quoted (e.g. Kyle Yarwood, haha), then the body is the actual quote

Reply:
In that case can we change it to image, quote, and speaker/person/author/etc for clarity?